SalaryService.md

```diff
@@ -0,0 +1,104 @@
+Salary Service
+====================
+
+Table of Contents
+_____________
+
+- [Summary](#summary)
+- [Request structure](#request-structure)
+- [Response structure](#response-structure)
+- [Versioning](#versioning)
+
+Summary
+-----------
+The Salary service returns salary info for a carotene id and, optionally, a postal or a cbsa code in either hourly or yearly amounts.
+
+e.g.: How much does a `Customer service representative` make per `year` in `Chicago`?
+
+The Salary service is available at `/core/salary`.
+
+Request Structure
+-----------
+
+Requests consist of:
+
+| Parameter | Require/Optional | Type | Description |
+|:----------|:-------|:-------|:-------|
+|`carotene_id`| Yes | string | A unique id|
+|`cbsa_code` | Optional | string |  Unique Identifiers for US Metropolitan Statistical areas. |
+|`postal_code` | Optional | string |  Optional. The post code, postal code, or ZIP Code of an address.|
```

Comment:
```suggestion
|`postal_code` | Optional | string ||
```

Reply:
I'd just omit the description here. If people don't know what postal codes are then they probably have much larger problems to deal with than figuring out how to use some API.

Reply:
Good point but leaving empty felt odd to me so added "a postal code"